tos/tests/test_models.py

```diff
@@ -1,26 +1,26 @@
+from django.contrib.auth import get_user_model
```

Comment:
Since Django 1.4 is still on the supported list at least for a little while longer, this project should continue to support.